docs/build-instructions.md

```diff
@@ -95,4 +92,5 @@ When installing, you must minimally select all Visual C++ components.
 5. Build the documentation.
     - `pipenv run mkdocs build`
 6. Access the generated documentation using your favorite browser.
-    - `cd ping-viewer && firefox index.html`
\ No newline at end of file
+    - `cd ping-viewer && firefox index.html`
+    - `cd ping-viewer && open -a "Firefox" index.html` on MacOS
```

Comment:
I believe that just `open` should do the job, not necessary firefox.

Reply:
I was being consistent with the existing suggestion, but I suppose open uses the default browser which takes account of the user's preference, so fair point.